addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -145,21 +145,23 @@
                                             </div>
                                         </td>
                                         <td class=" text-center h6" t-if="has_serial_number">
-                                            <span t-if="has_serial_number and (ml.lot_id or ml.lot_name)" t-esc="ml.lot_id.name or ml.lot_name" t-options="{'widget': 'barcode', 'humanreadable': 1, 'width': 400, 'height': 100, 'img_style': 'width:100%;height:35px;'}">
+                                            <span t-if="has_serial_number and (ml.lot_id or ml.lot_name) and ml.lot_id.valid_code_128" t-esc="ml.lot_id.name or ml.lot_name" t-options="{'widget': 'barcode', 'width': 400, 'height': 100, 'img_style': 'width:100%;height:35px;'}">
```

Comment:
Why remove `'humanreadable': 1`

Reply:
Added a separate span to print the text of the barcode so removed `humanreadable` attribute.